app/modules/events/event.ts

```diff
@@ -15,6 +15,10 @@ export function eventDataPatcher(event: RawEvent): FullEvent {
 }
 
 export function eventsDataPatcher(events: RawEvent[]): FullEvent[] {
+  if (!events) {
```

Comment:
We may be able to avoid this check by using `LoaderFunctionArgs` via the comment below.